core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/sklearn/SklearnMLOp.scala

```diff
@@ -0,0 +1,73 @@
+package edu.uci.ics.texera.workflow.operators.sklearn
+
+import com.fasterxml.jackson.annotation.{JsonIgnore, JsonProperty, JsonPropertyDescription}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort, PortIdentity}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{AttributeType, Schema}
+
+abstract class SklearnMLOp extends PythonOperatorDescriptor {
```

Comment:
this abstract base should be `SklearnMLOperatorDescriptor`. or you can keep it short as `SklearnMLOpDesc`.

Reply:
Fixed.